doc/user_guide/Reactive_Expressions.ipynb

```diff
@@ -478,6 +544,60 @@
     "rx(8.5).rx.pipe(str)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "376f5b3d-8bcd-4ad3-869d-cdee176e54d8",
+   "metadata": {},
+   "source": [
+    "#### `.rx.updating()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9334392f-61d0-4629-aae8-0c9a7935c6fa",
+   "metadata": {},
+   "source": [
+    "Returns a new expression that is True while the original expression is updating. Useful for performing some action while an expression is running.\n",
+    "\n",
+    "Here we create a simple expression that calls a `calculate` function which emulates a long running computation:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e132793e-b5ad-49bb-88d1-378a24451055",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expr = rx(1)\n",
+    "\n",
+    "def calculate(value):\n",
+    "    time.sleep(1)\n",
+    "    return value\n",
+    "\n",
+    "updating =  expr.rx.pipe(calculate).rx.updating()\n",
+    "\n",
+    "updating"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7eac3901-b9e6-4020-8f74-5817561a187e",
+   "metadata": {},
+   "source": [
+    "When we update the expression the `updating` expression will temporarily toggle to True and then reset:"
```

Comment:
Quite difficult since most of the time whatever action that is performing the update will be blocking which means you can't simultaneously perform throttling or debouncing on the event.

Reply:
Ok.

Reply:
Looking into it, since now I want this.

Reply:
Will only work for async callbacks and generators though.